examples/text_labels_and_annotations/arrow_demo.py

```diff
@@ -5,151 +5,83 @@
 
 Arrow drawing example for the new fancy_arrow facilities.
 
-Code contributed by: Rob Knight <rob@spot.colorado.edu>
-
-usage:
-
-  python arrow_demo.py realistic|full|sample|extreme
+Original author: Rob Knight <rob@spot.colorado.edu>
+"""
```

Comment:
```suggestion
"""
```

We don't cite people anymore..